.github/workflows/publish-release.yml

```diff
@@ -26,10 +26,6 @@ jobs:
           mv cli/build/distributions/cli-shadow-*.zip cli/build/distributions/cli.zip
```

Comment:
Is the '-*' here still required?

Reply:
Yes, because the dist-tar/zip keep the version suffix

Reply:
but I could also remove the suffix from the zip and tar files

Reply:
You removed the suffix there as well, do we need to update this `mv` statement?